languages/ruby/exercises/concept/booleans/.docs/after.md

```diff
@@ -0,0 +1,56 @@
+## True, False, and Nil
+
+- `true` and `false` are used to represent boolean logical states.
+  - They are singleton instances of the [`TrueClass`][true-class] and [`FalseClass`][false-class] objects.
+  - they may occur as literals in code, or as the result of logical (`&&`, `||`, `!`) or [comparison][comparable-class] (`<`, `>`, `==`) methods.
+- `nil` is a value to express _absence_ of an object.
+  - It is a singleton instance of the [`NilClass`][nil-class]
+  - It is often used for a default value
+    - The default value for class instance variables is `nil` unless otherwise assigned
+
+## Truthy, Falsey Logic
+
+- Only `false` and `nil` evaluate to _falsey_.
+- Everything else, objects and values, evaluate to _truthy_.
```

Comment:
What is a "value" in Ruby?

Reply:
Great question, since anything that comes from an evaluation would be an object.